src/components/Button/src/Button.vue

```diff
@@ -416,4 +489,28 @@ export default {
 .Button.align_space-between .InformationText {
 	margin-right: 8px;
 }
+
```

Comment:
if i can figure out how to put the icons in the `$s.MainText` section, i won't need these classes

Reply:
Ideally we wouldn't need any of these classes and the icons can be passed straight through the default slot. Please let us know what you have tried so far and why it's not working.